android/build.gradle

```diff
@@ -131,7 +131,9 @@ dependencies {
   // noinspection GradleDynamicVersion
   api 'com.facebook.react:react-native:+'
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation 'com.stripe:stripe-android:16.10.2'
+  implementation 'com.stripe:stripe-android:17.1.0'
+  implementation 'com.stripe:stripe-wechatpay:17.1.0'
+  implementation 'com.tencent.mm.opensdk:wechat-sdk-android-without-mta:6.7.0'
```

Comment:
The important thing is that `com.tencent.mm.opensdk:wechat-sdk-android-without-mta:6.7.0` is not included in the SDK gradle, only in the example project gradle. We need to make sure it is not a dependency for everyone using the SDK!

Reply:
as discussed offline, please move the following two dependencies to the actual example app
```
implementation 'com.stripe:stripe-wechatpay:17.1.0'
implementation 'com.tencent.mm.opensdk:wechat-sdk-android-without-mta:6.7.0'
```